### PR TITLE
Update dependencies to deal with CVEs

### PR DESCRIPTION
**Pull request by tomncooper**
### Type of change

- Bugfix

### Description

Bumps dependencies to deal with the following CVEs:

- CVE-2020-13956
- CVE-2021-36090
- CVE-2021-35515
- CVE-2021-35517

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -793,6 +794,12 @@
                 <artifactId>junit-platform-launcher</artifactId>
                 <version>${junit.platform.version}</version>
             </dependency>
+            <!-- transitive dep; override here to avoid buggy version -->
```
Comment (scholzj) [inline pom.xml:797]: Can you add which is using this as transitive dependency?

**Review by scholzj — APPROVED** [review]
One nit. LGTM otherwise.

File: pom.xml (hunk 1 above)
Comment (tomncooper) [reply pom.xml:797]: Good point. We should probably remove these overrides at some point and see if they are still relevant?

**Comment by tomncooper** [comment]
These test failures were due to timeouts. How do I re-run them?

**Review by tombentley — APPROVED** [review]

**Comment by scholzj** [comment]
@tombentley The original regression is still running. Why did you start a new one?

**Pull request merged**